README.md

```diff
@@ -1,102 +1,72 @@
-- [storj.pics](http://storj.pics) - Upload and share pictures using Storj 100% in browser. [[Example]](http://storj.pics/#/public/3c894b5bc1b2b8c8a69915c7/files/867cd8678ce8363eb6a38a28) [[Github]](https://github.com/nginnever/storj.pics)
-- [storjrb-backupcore](https://bitbucket.org/DaveahamLincoln/storjrb-backupcore) - A generic Ruby script that facilitates the automation of multiple Storj backups.
-- [storj-gui-client](https://github.com/lakewik/storj-gui-client) - GUI Client for Storj Network written in Python and PyQt4.
-- [Goobox community](https://github.com/GooBox/goobox-community-gui) - A desktop file synchronization client for Storj.
-
-# Tutorials & Examples
-- [Quickstart](https://docs.storj.io/) - Steps thought creating an acccount, and using CLI tools.
-- [Tutorials](https://storj.github.io/core/) - Full documentation for Storj Core, with many example scripts.
-- [Storj Node Heroku Example in Node.js](https://github.com/Storj/storj-node-heroku-example) - Authenticate, create key pairs, create buckets, and upload and download a file.
-- [Working with the Storj API](https://docs.google.com/document/d/1ehsSHtwnwC-LSgygxYGFuWoCx1DuhA2-XbDw64nggNY/edit?usp=sharing) - Order of method calls for uploading and downloading with the Storj API.
-- [FAQ](https://storj.io/faq.html) - Frequently Asked Questions.
-
-# Whitepapers
-- [Storj Whitepaper v2](https://storj.io/storj.pdf) - Published December 15, 2016.
-- [Storj Whitepaper v1](https://storj.io/storj2014.pdf) - Published December 15, 2014.
-
-# Community
-- [Community Chat](https://community.storj.io/) - Open community chat. Our most active discussion area.
-- [Community Page](https://storj.io/community.html) - Links to newsletter, chat, social media, and events.  
-- [Discord](https://discord.gg/S6KTchS) - Community discord server with chat and voice chat. 
-
-# Contribute to Storj
-- [Storj on Github](https://github.com/storj) - All Storj repositories in one place.
-- [Give Feedback](https://wantoo.io/storj-product-feedback/) - Suggest new product ideas or upvoting existing ones.
-- [Write Guides](https://storj.io/get-paid-to-write.html) - Get paid to write guides and tutorials about our platform.
-- [Submit Code](https://storj.io/developers.html) - Submit pull requests on our public repositories on GitHub.
+## Services based on / with support of Storj 
+
+* [Fastly](https://fastly.com) - Content delivery network with Storj support ([docs](https://docs.fastly.com/en/guides/storj-dcs-object-storage))
+* [Filebase](https://filebase.com/) - S3 compatible Object Storage. One of the supported storage backend is Storj ([docs](https://docs.filebase.com/what-is-filebase/our-ecosystem/storage-networks/storj#how-do-i-store-data-on-storj-through-filebase))
+* [Opensea](https://opensea.io/) -- NFT marketplace. Supports hosting NFT resources on Storj ([howto](https://docs.storj.io/dcs/how-tos/nft-storage))
+
```

Comment:
Arent we missing Pixelexperience here that use us to distribute their roms?

Reply:
Sure, I added them.

Long-term we may need to differentiate between the services who use Storj under the hood and the services who offer Storj as an option. But as I am writing this I just realized that how thin is this difference ;-)